trigger/version.go

```diff
@@ -107,9 +108,21 @@ func (t *versionTriggers) makeData(sub *event.Subscription, pastTenseOverride st
 		projectName = utility.FromStringPtr(api.ProjectIdentifier)
 	}
 
-	collectiveStatus, err := patch.CollectiveStatus(t.version.Id)
-	if err != nil {
-		return nil, errors.Wrap(err, "getting collective status for patch")
+	var status string
```

Comment:
if it is not a patch requester, the status needs to be set to `t.data.Status`

Reply:
Ah yeah, I forgot to push the commit with the actual else statement here.